pyproject.toml

```diff
@@ -0,0 +1,53 @@
+[project]
+name = "animalai"
+version = "4.0.0"
+description = "Animal AI Python API"
+license = "MIT"
+readme = "README.md"
+authors = [
+    { name = "Animal-AI Team", email = "kindsofintelligence.cfi@gmail.com" },
+    { name = "Matt Crosby" },
+    { name = "Ibrahim Alhas" },
+    { name = "Konstantinos Voudouris" },
+    { name = "Wout Schellaert" },
+    { name = "Joel Holmes" },
+    { name = "Ben Beyret" },
+]
+classifiers = [
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "Topic :: Scientific/Engineering :: Artificial Intelligence",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Operating System :: Microsoft :: Windows :: Windows 10",
+    "Operating System :: Microsoft :: Windows :: Windows 11",
+    "Operating System :: MacOS :: MacOS X",
+    "Operating System :: POSIX :: Linux",
+]
+requires-python = ">=3.8, <3.10"
+dependencies = [
+    "mlagents_envs==0.30.0",
```

Comment:
A few factors are in play which dictates the specified version of ml-agents we are using, mainly that ml-agents is deprecated and new versions are rare (there is a newer version but we should'nt expect massive changes or improvements since funding for the project is withheld indefinitely).

Python versioning is strict for 0.30.0 and for 1.0.0 (new version of ml-agents).  Also, there were a few dependency issues, namely numpy, protobuf, which required to be specified as listed on ml-agents documentation. However, these can change if/when we upgrade to ml-agents 1.0.0 after evaluation and testing.